app/templates/reading_list.html

```diff
@@ -0,0 +1,65 @@
+{% extends "base_root.html" %}
+
+{% block navlinks %}
+  {{ super() }}
+{% endblock %}
+
+{% block main_body %}
+
+<div class="container">
+<div id="readinglist">
+    <div class="col-lg-10 ml-auto mr-auto">
+    	<h2 class="text-left title text-black">Your Book List</h2>
+        <div class="thumbnail">
```

Comment:
The `thumbnail` class has not purpose here

Reply:
If your idea was to give each row a specific class-name to later identify all "bookmark-row" in the DOM (e.g. with jQuery) adding a class without css rules is a good idea. I would be very careful though to choose a verbose and expressive name (e.g.  "bookmark-row") over a general name ( "thumbnail") to avoid confusion and side effects (e.g. if a "thumbnail" class exists in bootstrap).